p2p/starknet/handlers.go

```diff
@@ -71,17 +76,20 @@ func streamHandler[ReqT proto.Message](ctx context.Context, stream network.Strea
 
 	response, err := reqHandler(req.(ReqT))
```

Comment:
What do you think about renaming "response" to "requestIterator" or something?

Reply:
This may be non-standard, but renaming yield in processIterationRequest to processRquest might also make it easier to understand at first glance

Reply:
> What do you think about renaming "response" to "requestIterator" or something?

I renamed it to "responseIterator" since it's iterator over response.

Reply:
> This may be non-standard, but renaming yield in processIterationRequest to processRequest might also make it easier to understand at first glance

afaik `yield` is standard name for generators (python, js, php and few others have this statement keyword). If that's ok I'll keep it for a while if there are more complains about naming we will rename it